src/mcpwm.rs

```diff
@@ -122,25 +122,171 @@ impl From<CounterMode> for mcpwm_counter_type_t {
+    ///               |   .               >.   |< fed
+    /// MCPWMXB out   |   .                .   |
+    ///               |   .                .   |
+    /// --------------.   .                .   -----------------
+    ///               .   .                .   .
     ActiveHigh { red: u16, fed: u16 },
 
     /// MCPWM_ACTIVE_LOW_MODE
+    ///
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .--------------------.   .
+    ///               |   .                |   .
+    /// MCPWMXA in    |   .                |   .
+    ///               |   .                |   .
+    /// ---------------   .                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    /// ------------------.                .--------------------
+    ///          red >.   |<               |   .
+    /// MCPWMXA out   .   |                |   .
+    ///               .   |                |   .
+    ///               .   ------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    /// --------------.   .                .   .----------------
+    ///               |   .               >.   |< fed
+    /// MCPWMXB out   |   .                .   |
+    ///               |   .                .   |
+    ///               --------------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
     ActiveLow { red: u16, fed: u16 },
 
-    /// MCPWM_ACTIVE_HIGH_COMPLIMENT_MODE
+    // TODO: Is this actually true? --------
+    //                                      |
+    //                                      v
+    /// MCPWM_ACTIVE_HIGH_COMPLIMENT_MODE - The most common deadtime mode
+    ///
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .--------------------.   .
+    ///               |   .                |   .
+    /// MCPWMXA in    |   .                |   .
+    ///               |   .                |   .
+    /// ---------------   .                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    ///               .   .----------------.   .
+    ///          red >.   |<               |   .
+    /// MCPWMXA out   .   |                |   .
+    ///               .   |                |   .
+    /// -------------------                ---------------------
+    ///               .   .                .   .
+    ///               .   .                .   .
+    /// --------------.   .                .   .----------------
+    ///               |   .               >.   |< fed
+    /// MCPWMXB out   |   .                .   |
+    ///               |   .                .   |
+    ///               --------------------------
```

Comment:
After reading both the [technical_reference_manual](https://www.espressif.com/sites/default/files/documentation/esp32-s3_technical_reference_manual_en.pdf#page=923) and the [API reference](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/api-reference/peripherals/mcpwm.html#_CPPv421mcpwm_deadtime_type_t) I still can not wrap my head around what happens with the MCPWMXB out signal. Does it completely ignore MCPWMXB in, or is it `AND` with it? In other words, is `operator.set_duty_b()` completely useless once a deadtime is set?

Reply:
And BTW I still have not received the S3 I ordered, so I am unable to test it...

Reply:
> Does it completely ignore MCPWMXB in, or is it AND with it? In other words, is operator.set_duty_b() completely useless once a deadtime is set?

This is my interpretation of the docs, but I'd need to get around to setting up the hardware too.

> And BTW I still have not received the S3 I ordered, so I am unable to test it...

Could you drop me an email (on my gh profile), or DM me on matrix?

Reply:
> In other words, is operator.set_duty_b() completely useless once a deadtime is set?

Tested and confirmed with `DeadtimeConfig::ActiveHighComplement` on ESP32(no suffix). Will try to test some other modes as well.